src/main/java/com/epam/ms/util/UserServiceMediator.java

```diff
@@ -0,0 +1,49 @@
+package com.epam.ms.util;
+
+import com.epam.ms.util.dto.User;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.client.WebClient;
+
+import java.util.List;
+
+/**
+ * The class is used to send request to the user-service
+ */
+@Component
+public class UserServiceMediator {
+    private static final String HOST_URL = "http://localhost:%s";
+    private static final String GET_USER_URL = "/users/%s";
+    private static final String GET_USERS_URL = "/users";
+    private static final Integer USER_SERVICE_PORT = 8080;
+
+    public User getUser(String id) {
+        WebClient client = WebClient.builder()
```

Comment:
you can instantiate you WebClient once somewhere in constructor, for instance

Reply:
Great idea, thank you!